static/src/js/reducers/query.js

```diff
@@ -18,7 +21,7 @@ const initialState = {
     pageNum: 1,
     spatial: {},
     temporal: {},
-    sortKey: ['-usage_score']
+    sortKey: [collectionSearchResultsSortKey || '-score']
```

Comment:
This doesn't need a default value added, it can just be the `collectionSearchResultsSortKey`